# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1512664450 18000
#      Thu Dec 07 11:34:10 2017 -0500
# Node ID 51a1d6c84d0c0e18a69c3e735c15e11273fe301a
# Parent  56d1e4d9975e869bbc9692e6aff6f34c1cbcfe45
* lex.h: Include symscope.h for clang on OS X.

diff --git a/libinterp/parse-tree/lex.h b/libinterp/parse-tree/lex.h
--- a/libinterp/parse-tree/lex.h
+++ b/libinterp/parse-tree/lex.h
@@ -28,22 +28,22 @@ along with Octave; see the file COPYING.
 #include <deque>
 #include <limits>
 #include <list>
 #include <set>
 #include <stack>
 
 #include "comment-list.h"
 #include "input.h"
+#include "symscope.h"
 #include "token.h"
 
 namespace octave
 {
   class interpreter;
-  class symbol_scope;
 
   // Is the given string a keyword?
   extern bool is_keyword (const std::string& s);
 
   // For communication between the lexer and parser.
 
   class
   lexical_feedback
